boards/x86/up_squared/doc/index.rst

```diff
@@ -251,6 +251,40 @@ Booting UP Squared
 #. When the boot process completes, you have finished booting the
    Zephyr application image.
 
+Integration Testing With Twister
+================================
+
+The up_squared hardware also has integration for testing using the twister tool.
+The script behind ``--west-flash`` will copy the ``*.efi`` file to the pxe server,
+then restart up_squared board via smartPDU.
+
+  .. code-block:: console
+
+    $ twister -p up_squared --device-testing --device-serial <uart serial port> --west-flash=<helper script> -v -T <test suite path>  -v
+
+The content inside helper script:
+
+  .. code-block:: console
+
+    #!/bin/bash
```

Comment:
this is a test setup that is external to zephyr and is really not related to operating the board and can be done in many different ways, not sure why it is being added to the board documentation. Maybe a lab/ test environment setup section in the documentation can be started with such information collected for various hardware platforms under test and ways to test them instead of making this part of the board documentation and share across boards of the same class.

Reply:
Thanks to note that, i will close this pr and add this message to internal doc.